packages/app/components/profile/Profile.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FunctionComponent } from 'react';
+import { useQuery } from '@apollo/react-hooks';
+
+import { USER } from './queries';
+
+interface QueryProps {
+  user: {
+    picture: string;
+    email: string;
+    last_login: string;
+  }[];
+}
+
+export const Profile: FunctionComponent = () => {
+  const { data, error, loading } = useQuery<QueryProps>(USER);
```

Comment:
Should we move that to a QueryType file ? and where ?

Reply:
Soit on créé un fichier types.ts par composant, soit on peut le mettre dans le fichier queries.ts ou mutations.ts